src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,330 @@
+                    continue;
+                }
+                addEventTask(separatedKeyWordAndContent[1], tasks, dataFile);
+            } else if (keyword.equals("mark") || keyword.equals("unmark")) {
+                if (userInput.split(" ").length != 2) {
+                    System.out.println(INSTRUCTION + "\n"
+                            + keyword + ": Number");
+                } else {
+                    changeTaskStatus(tasks, separatedKeyWordAndContent);
+                }
+            } else {
+                System.out.println(MEANINGLESS_SENTENCE_AFTER_KEYWORD);
+            }
+
+        }
+    }
+
+    private static void createHorizontalLine() {
+        System.out.println(HorizontalLine);
+    }
+
+    private static String getInput(Scanner in) {
+        String userInput;
+        createHorizontalLine();
+        System.out.println();
+        userInput = in.nextLine();
+        createHorizontalLine();
+        return userInput;
+    }
+
+    private static void showWelcomeMessage() {
+        createHorizontalLine();
+        System.out.println(HELLO_IM_DUKE);
+        System.out.println(KEYWORD_TO_SEE_THE_INSTRUCTIONS);
+        System.out.println(ASKING_MESSAGE);
+    }
+
+    private static void showHelpMessage() {
+        System.out.println("This is the list of our commands\n");
+        System.out.println(HELP_USAGE);
+        System.out.println(BYE_USAGE);
+        System.out.println(LIST_USAGE);
+        System.out.println(MARK_USAGE);
+        System.out.println(MARK_FORMAT);
+        System.out.println(UNMARK_USAGE);
+        System.out.println(UNMARK_FORMAT);
+        System.out.println(TODO_USAGE);
+        System.out.println(TODO_FORMAT);
+        System.out.println(DEADLINE_USAGE);
+        System.out.println(DEADLINE_FORMAT);
+        System.out.println(EVENT_USAGE);
+        System.out.println(EVENT_FORMAT);
+        System.out.println(DELETE_USAGE);
+        System.out.println(DELETE_FORMAT);
+    }
+
+    private static boolean doesIndexOutOfBoundsOccur(String[] stringArray, int index, String outputMessage) {
+        try {
+            String test = stringArray[index];
+            return false;
+        } catch (ArrayIndexOutOfBoundsException error) {
+            System.out.println(outputMessage);
+            return true;
+        }
+    }
+
+    private static void addTodoTask(String content, ArrayList<Task> tasks, FileClass dataFile) throws EmptyDescription {
+        if (content.trim().isEmpty()) {
+            throw new EmptyDescription();
+        }
+        Task task = new ToDo(content);
+        tasks.add(task);
+        System.out.println(ADDING_TASK);
+        System.out.println("[T][] " + content);
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        String inputToDataFile = "T | " + convertMarkingStatusToNumber(task) + " | " + task.getContent() + "\n";
+        appendTaskToDataFile(dataFile, inputToDataFile);
```

Comment:
Instead of repeating this portion of similar code for both `todo`, `deadline` and `event`, you might want to consider applying the polymorphism concept here. In simpler words, each type of task should know how to print themselves - so in your adding task logic here, you only need to call `System.out.println(task.toString())` or something similar to this.